VEDA_OS01/tests/test_views.py

```diff
@@ -48,8 +50,6 @@ def test_transcript_credentials_unauthorized(self):
         )
 
         response_status_code = response.status_code
-        response = json.loads(response.content)
-
         self.assertEqual(response_status_code, status.HTTP_401_UNAUTHORIZED)
```

Comment:
I think we can use `response.status_code` in assert, but I guess it's not in your PR scope, was missed in previous PR I guess.